sdk/bom/azure-sdk-bom.xml

```diff
@@ -0,0 +1,131 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-sdk-bom</artifactId>
```

Comment:
Please do a little bit of research on the common naming patterns of these BOMs in other libraries - this was just the name I randomly selected when I mocked up this BOM.

Reply:
Looking at a few other project there appears to be a similar naming pattern of `<project reference>-bom`, I'm going to leave the artifact name as `azure-sdk-bom`.